index.html

```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
+        </div>
+        <div class="tonic-btn">
+            <button type="button"> See Project</button>
+        </div>
+    </div>
+        <div class="tonic-container">
+            <div><img src="images/tonic2.png" alt="tonic"></div>
+            <div class="tonic-heading">Tonic</div>
+            <div class="canopy-container">
+                <div class="canopy-header">CANOPY</div>
+                <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+                <div class="canopy-text">Back End Dev</div>
+                <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+                <div class="canopy-text">2015</div>
+            </div>
+            <div class="tonic-parag">
+                <p>
+                    A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required.
+                </p>
+            </div>
+            <div class="tonic-icons-container">
+                <ul>
+                    <li><img src="images/html-tag.png" alt="html-tag"></li>
+                    <li><img src="images/css-tag.png" alt="css-tag"></li> 
+                    <li><img src="images/javascript-tag.png" alt="javascript-tag"></li> 
+               </ul>
+            </div>
+            <div class="tonic-btn">
+                <button type="button"> See Project</button>
+            </div>
+        </div>
+            <div class="tonic-container">
+                <div><img src="images/multi-post-stories2.png" alt="multi-post-stories2"></div>
+                <div class="tonic-heading">Multi-Post Stories</div>
+                <div class="canopy-container">
+                    <div class="canopy-header">CANOPY</div>
+                    <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+                    <div class="canopy-text">Back End Dev</div>
+                    <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+                    <div class="canopy-text">2015</div>
+                </div>
+                <div class="tonic-parag">
+                    <p>
+                        A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required.
+                    </p>
+                </div>
+                <div class="tonic-icons-container">
+                    <ul>
+                        <li><img src="images/html-tag.png" alt="html-tag"></li>
+                        <li><img src="images/css-tag.png" alt="css-tag"></li> 
+                        <li><img src="images/javascript-tag.png" alt="javascript-tag"></li> 
+                   </ul>
+                </div>
+                <div class="tonic-btn">
+                    <button type="button"> See Project</button>
+              </div>
+            </div>
+                
+</section>
+<footer class="footer-container">
+    <h1>
+        About <br>
+        Myself
+    </h1>
```

Comment:
- [ ] Kindly remove the `footer` tag from here and use the `section` tag. "About Myself" section (line 151- line 201) should be wrapped in a `section` tag. :+1: